src/app/pages/datasets/components/dataset-capacity-management-card/space-management-chart/space-management-chart.component.ts

```diff
@@ -57,16 +57,17 @@ export class SpaceManagementChartComponent implements OnChanges {
   }
 
   private updateChartData(): void {
+    // TODO: Used by Snapshots was removed.
```

Comment:
What means TODO?

Reply:
@denysbutenko - it's a reminder that some data was removed